test/studies/prk/stencil/EXECOPTS

```diff
@@ -0,0 +1,2 @@
+--iterations=10 --order=1000 --validate
```

Comment:
Similarly, are these problem sizes that will finish reasonably quickly for correctness tests (under 5 seconds)? I have no idea, just checking because of the similiar COMPOPTS things.